content/docs/builders.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Builders"
+weight: 101
+menu: "main"
+---
+
+# Builders
+In the Getting Started tutorial, you used the `pack` CLI and the Paketo base `builder` to create a runnable image with your application. This section explains what Paketo `builders` are, and how they fit into the Paketo ecosystem.
+
+## What is a Paketo builder?
```

Comment:
This section should be generic to CNB. So something like "What is a Builder" and the explanations underneath updated to remove "paketo"